be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -26,6 +26,7 @@
 #include "vec/exprs/vexpr_context.h"
 #include "vec/utils/template_helpers.hpp"
 #include "vec/utils/util.hpp"
+#include "gutil/strings/substitute.h"
```

Comment:
should use BE code formater to format the code.

Reply:
done